### ISSUE TITLE
[Fixed in GH-6052] Hidden Visual Focus on Section2A and 2B banners

### ISSUE THREAD

**Issue by developpeurweb** [SAP/cloud-commerce-spartacus-storefront#4295]
![Screen Shot 2019-08-28 at 10 18 21 PM](https://user-images.githubusercontent.com/15113219/63905407-2e69fa80-c9e2-11e9-8e4f-2367eba54dd9.png)

Element receives visual focus but the image hides it partially

**Comment by developpeurweb** [comment]
**For QA:**

- To test in Safari please refer to [these instructions](https://github.com/SAP/cloud-commerce-spartacus-storefront/issues/5041#issuecomment-558883497).
- This got fixed in ticket GH-6052
- Theme `calydon` should be enabled to see changes.
- Visible focus outline overlapping adjacent ui elements may happen and may not be an issue, if in doubt please open a ticket per element or reach out to UI developer.
- Please separate visible focus issues (CSS: element without outline) from tabbing issues (HTML/TS: element not reachable with keyboard).

![Screen Shot 2020-01-31 at 4 48 38 PM](https://user-images.githubusercontent.com/15113219/73576894-90bf9f00-4449-11ea-8450-ee6936c58068.png)
